test/Test/Data/Array.purs

```diff
@@ -467,6 +467,25 @@ testArray = do
     , [4,0,0,1,25,36,458,5842,23757]
     ]
 
+  log "sliding"
+  assert $ A.sliding [ 1, 2, 3, 4, 5 ] == [ (Tuple 1 2), (Tuple 2 3), (Tuple 3 4), (Tuple 4 5) ]
```

Comment:
Can you add a test for `A.sliding [] == []`?

Reply:
done